core/modules/layout/layout.admin.inc

```diff
@@ -2297,3 +2297,66 @@ function layout_ajax_form_update($form, $form_state) {
 
   return array('#type' => 'ajax', '#commands' => $commands);
 }
+
+/**
+ * Menu callback; Show the region styles form.
+ */
+function layout_configure_region_page($form, &$form_state, Layout $layout, $renderer_name, $region_name, $block_key = NULL) {
+  $form_state['layout'] = $layout;
+  $form_state['region_name'] = $region_name;
+  $layout_info = layout_get_layout_info($layout->layout);
+  if (!array_key_exists($region_name, $layout_info['regions'])) {
+    return MENU_NOT_FOUND;
+  }
+
+  $form['#tree'] = TRUE;
+
+  $options = array(
+    '' => 'No inner wrapper',
+    'div' => 'DIV',
+    'navgit' => 'NAV',
```

Comment:
`navgit` => `nav`.

I suppose it's possible that an entire region could be a `nav`, e.g. a mega-menu type situation where the menu could be made up of multiple blocks, like both a menu and a view.